CMakeLists.txt

```diff
@@ -216,20 +230,21 @@ add_custom_target(utrecht_tiles DEPENDS ${VALHALLA_SOURCE_DIR}/test/data/utrecht
 
 set_target_properties(logging PROPERTIES COMPILE_DEFINITIONS LOGGING_LEVEL_ALL)
 
-if(ENABLE_DATA_TOOLS)
-  add_dependencies(mapmatch utrecht_tiles)
-endif()
-
 # Test run targets
 foreach(test ${tests} ${cost_tests})
   add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test}.log
-    COMMAND LOCPATH=locales /bin/bash -c -o pipefail "${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test} | tee ${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test}.log"
+    COMMAND LOCPATH=locales /bin/bash -c "${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test} >& ${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test}.log && echo PASS ${test} || (exit=$? ; echo FAIL with exit code $exit ; cat ${CMAKE_CURRENT_BINARY_DIR}/unit_tests/${test}.log ; exit $exit)"
```

Comment:
cool i see you stored the exit code there. that should work! would you mind adding the color coding i suggested. in between all the dependencies messages and stuff it makes the status really stand out